packages/core/strapi/lib/core/registries/controllers.d.ts

```diff
@@ -0,0 +1,7 @@
+import { Strapi } from '../../';
+
+export type Controller = {
+  [key: string]: (...args: any) => any;
```

Comment:
Not sure if this type is really useful in this state? We could at least list the utils we pass in the proto of controllers? WDYT?